tests/lib/ignored-paths.js

```diff
@@ -158,7 +158,11 @@ describe("IgnoredPaths", () => {
                 ignorePattern
             });
 
-            assert.ok(ignorePattern.every(pattern => getIgnoreRules(ignoredPaths).filter(rule => (rule.pattern === pattern)).length > 0));
+            assert.ok(
+                ignorePattern.every(pattern =>
+                    getIgnoreRules(ignoredPaths).some(rule => rule.pattern === pattern)
```

Comment:
@platinumazure In addition to moving this onto multiple lines as you suggested, I also changed `.filter(foo).length > 0` to `.some(foo)`